docs/release-notes/8.0.33-25.md

```diff
@@ -0,0 +1,108 @@
+# Percona Server for MySQL 8.0.33-25 (2023-06-05)
+
+<style>
+    table {
+        width=50%’;
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+</style>
+
+| Release date | June 05, 2023 |
+|--- | --- |
+| Install instructions | [Install Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/installation.html) |
+| Upgrade instructions | [Percona Server for MySQL in-place upgrading guide: from 5.7 to 8.0](https://docs.percona.com/percona-server/8.0/upgrading_guide.html) |
+
+[Percona Server for MySQL 8.0.33-25](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.33 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-33.html) in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a freely available, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior and optimized performance, greater scalability, and availability, enhanced backups, increased visibility and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide better performance and concurrency for their most demanding workloads.
+
+## Release highlights
+
+* This release adds the following MyRocks variables:
+
+  * `rocksdb_bulk_load_use_sst_partitioner`
+
+  * `rocksdb_use_hyper_clock_cache`
+
+  * `rocksdb_converter_record_cached_length`
+
+  * `rocksdb_alter_table_comment_inplace`
+
+  * `rocksdb_charge_memory`
+
+  * `rocksdb_column_default_value_as_expression`
+
+  * `rocksdb_corrupt_data_action`
+
+  * `rocksdb_disable_instant_ddl`
+
+  * `rocksdb_enable_delete_range_for_drop_index`
+
+  * `rocksdb_partial_index_blind_delete`
+
+  * `rocksdb_protection_bytes_per_key`
+
+  * `rocksdb_use_write_buffer_manager`
+
+* This release adds a new MyRocks Information Schema Table `ROCKSDB_LIVE_FILES_METADATA`.
+
+* This release removes `rocksdb_instant_ddl` variable. Use `rocksdb_disable_instant_ddl` instead. 
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.33 and included in Percona Server for MySQL are the following:
+
+* The `audit_log_database` server system variable permits specifying other databases in the global schema namespace at server startup. The default setting for table storage is `mysql` system database.
```

Comment:
This needs to be completely removed - this feature exist only in Oracle's Enterprise audit_log plugin v2.
Percons Server in this version (8.0.33-25) still does not have this plugin.

Reply:
removed